### PR TITLE
Added global version callout

### PR DESCRIPTION
**Pull request by nataliefiann**
## What are you changing in this pull request and why?

I have created this PR to add a global VersionCallout component that allows us to set a version callout for the snapshot_meta_column_names and Unit tests doc

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-nat-add-version-callout-dbt-labs.vercel.app/docs/build/unit-tests
- https://docs-getdbt-com-git-nat-add-version-callout-dbt-labs.vercel.app/reference/resource-configs/snapshot_meta_column_names

### CODE REVIEW

File: website/docs/docs/build/unit-tests.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ keywords:
   - unit test, unit tests, unit testing, dag
 ---
 
+<VersionCallout version="1.9" />
+
 :::note 
 
 Unit testing functionality is available in [dbt Cloud Release Tracks](/docs/dbt-versions/cloud-release-tracks) or dbt Core v1.8+
```
Comment (mirnawong1) [inline website/docs/docs/build/unit-tests.md]: @nataliefiann should you remove the line 13 callout in lieu of the component?

File: website/docs/docs/build/unit-tests.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ keywords:
   - unit test, unit tests, unit testing, dag
 ---
 
+<VersionCallout version="1.9" />
```
Comment (mirnawong1) [inline website/docs/docs/build/unit-tests.md]: should this be 1.8 instead since unit tests are available from 1.8 and higher?

Comment (nataliefiann) [inline website/docs/docs/build/unit-tests.md]:
```suggestion
<VersionCallout version="1.8" />
```

File: website/docs/reference/resource-configs/snapshot_meta_column_names.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ default_value: {"dbt_valid_from": "dbt_valid_from", "dbt_valid_to": "dbt_valid_t
 id: "snapshot_meta_column_names"
 ---
 
-Available in dbt Core v1.9+. Select v1.9 or newer from the version dropdown to view the configs. Try it now in the [dbt Cloud "Latest" release track](/docs/dbt-versions/cloud-release-tracks).
+<VersionCallout version="1.9" />
```
Comment (nataliefiann) [inline website/docs/reference/resource-configs/snapshot_meta_column_names.md:9]:
```suggestion
<VersionCallout version="1.8" />
```

File: website/docs/reference/resource-configs/snapshot_meta_column_names.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ default_value: {"dbt_valid_from": "dbt_valid_from", "dbt_valid_to": "dbt_valid_t
 id: "snapshot_meta_column_names"
 ---
 
-Available in dbt Core v1.9+. Select v1.9 or newer from the version dropdown to view the configs. Try it now in the [dbt Cloud "Latest" release track](/docs/dbt-versions/cloud-release-tracks).
+<VersionCallout version="1.8" />
```
Comment (nataliefiann) [inline website/docs/reference/resource-configs/snapshot_meta_column_names.md]:
```suggestion
<VersionCallout version="1.9" />
```

File: website/docs/docs/build/unit-tests.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ keywords:
   - unit test, unit tests, unit testing, dag
 ---
 
+<VersionCallout version="1.8" />
+
 :::note
```
Comment (nataliefiann) [inline website/docs/docs/build/unit-tests.md]:
```suggestion

```

File: website/docs/docs/build/unit-tests.md [hunk 6]
```diff
@@ -8,7 +8,8 @@ keywords:
   - unit test, unit tests, unit testing, dag
 ---
 
-:::note 
+<VersionCallout version="1.8" />
+
 
 Unit testing functionality is available in [dbt Cloud Release Tracks](/docs/dbt-versions/cloud-release-tracks) or dbt Core v1.8+
```
Comment (nataliefiann) [inline website/docs/docs/build/unit-tests.md]:
```suggestion

```

File: website/docs/docs/build/unit-tests.md [hunk 7]
```diff
@@ -8,9 +8,9 @@ keywords:
   - unit test, unit tests, unit testing, dag
 ---
 
-:::note 
+<VersionCallout version="1.8" />
+
 
-Unit testing functionality is available in [dbt Cloud Release Tracks](/docs/dbt-versions/cloud-release-tracks) or dbt Core v1.8+
 
 :::
```
Comment (nataliefiann) [inline website/docs/docs/build/unit-tests.md]:
```suggestion

```

**Comment by mirnawong1** [comment]
lgtm!

**Pull request merged**